portafly/src/i18n/locales/en/product.yml

```diff
@@ -54,7 +54,7 @@ edit:
   #-------# save_button: available in shared.yml
   #-------# cancel_button: available in shared.yml
   # Validation errors
-  required_name: "<i>Name</i> is empty. You must specify a name for the product."
+  required_name: <i>Name</i> is empty. You must specify a name for the product.
```

Comment:
Why adding quotes in [line 33](https://github.com/3scale/porta/pull/2158/files#diff-fa4031e06a85bb2ff7ea9a7b088b89a1R33), but deleting them here?
Quotes are only needed for interpolation, but not for formatting?

Reply:
Correct, quotes are only needed for interpolation. In order to use formatting, it's necessary to use the `Trans` component, otherwise it's just plain text.

Reply:
I added 2 examples in order to illustrate this:
- Strings: https://github.com/3scale/porta/pull/2158/files#diff-2bf1f6e1e9a3b913f841030ab4e1d713R8-R9
- React: https://github.com/3scale/porta/pull/2158/files#diff-b01dc0b374f13f56cfecf892a70856afR173-R174